packages/charrua-core/charrua-core.0.5/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/mirage/charrua-core/releases/download/v0.5/charrua-core-0.5.tbz"
```

Comment:
@yomimono sorry for the noise, please go on with the change and I'll create the other ones retroactively since github lets me upload a file to an existing tag/release.